scripts/ci/install_deps.sh

```diff
@@ -3,23 +3,30 @@
 # exit when any command fails
 set -ex
 
+
 if [[ $TEST_SUITE = "Kubernetes"  ]]; then
-  # conntrack is required by Minikube
-  sudo apt-get update
-  sudo apt-get install -y conntrack
 
   # Download kubectl, which is a requirement for using minikube.
   curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v"${KUBERNETES_VERSION}"/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 
-  # Download Helm
+  sudo apt-get update
+
+  # Make test dir to store test results
+  mkdir -p "${TEST_RESULTS}"
+
+  # conntrack is required by Minikube
+  sudo apt-get install -y conntrack
+
+  curl -sSL https://github.com/gotestyourself/gotestsum/releases/download/v"${GOTESTSUM_VERSION}"/gotestsum_"${GOTESTSUM_VERSION}"_linux_amd64.tar.gz | sudo tar -xz -C /usr/local/bin gotestsum
```

Comment:
Nit: You have the same command in the _else_ block. Maybe it would be better to move it outside the _if/else_?